http/src/main/scala/org/broadinstitute/dsde/workbench/leonardo/http/service/AppServiceInterp.scala

```diff
@@ -594,12 +599,15 @@ final class LeoAppServiceInterp[F[_]: Parallel](config: AppServiceConfig,
       )
       app <- appQuery.save(saveApp, Some(ctx.traceId)).transaction
 
+      landingZoneResources <- getLandingZoneResources(workspaceDesc.spendProfile, userToken)
```

Comment:
Thinking we can move this up, closer to the other WSM call on L479?

That way if we fail to get the LZ we can fail faster, before inserting app records to the DB.

Reply:
Oh, oops! Moved and added checking for cloud provider-